### ISSUE TITLE
Suggestion: include "gg" to go move to the beginning of a document

### ISSUE THREAD

**Issue by suchtie** [rtorr/vim-cheat-sheet#107]
Of course you can use `1G` but `gg`, in the same vein as `yy` or `dd`, is much faster as it uses only 2 keypresses. I frequently find myself jumping to the top of a document so this has helped me a lot so far.

**Comment by rtorr** [comment]
@suchtie want to make s pull request? Great idea :)

**Comment by suchtie** [comment]
I would if I knew how to properly use git. :/ I manage repos just for myself so I only write directly to master. That's why I made a suggestion thread in the first place.

Well, I have some time on my hands anyway, why not spend it learning something useful?  Time to read that nice git wikibook again, I guess.

**Comment by tkraak** [comment]
@suchtie ... please get in touch ... I'd be glad to talk you thru Git & Jekyll.

**Comment by tkraak** [comment]
I'll also spell out the steps here later tonight for everybody else.

**Comment by rtorr** [comment]
@suchtie this repo is friendly to first time pulls requests. I'll help you through it. Let me know if you have issues with the readme or GitHub! :)

**Comment by tkraak** [comment]
@suchtie I've initiated the pull requests for you.

Here is a high-level overview of the steps involved:

* fork this repository (make a personal copy into your GitHub account)
* clone your fork (download the forked repo to your machine)
* create a new branch off gh-pages and switch to it
* open [index.html and _config.yml](https://github.com/rtorr/vim-cheat-sheet/pull/108/files) and add to it appropriately
* commit your changes
* push your changes
* open a pull request from your fork

As @rtorr mentioned, this is a beginner-friendly repository ... never hesitate to ask for help!

**Comment by rtorr** [comment]
@tkraak awesome!

**Comment by rtorr** [comment]
@tkraak should I wait on those pull requests?

**Comment by rtorr** [comment]
@tkraak also my only suggestion is to do a pull request on master and not gh-pages

**Comment by rtorr** [comment]
@tkraak so it looks like you made changes on the gh-pages branch on your local. Both should be master, that way there isnt a huge merge.

**Comment by tkraak** [comment]
@rtorr ... thanks! So you are saying I should never touch gh-pages at all?

**Comment by rtorr** [comment]
@tkraak as of right now, I issue pull requests from master to gh-pages. When you clone the repository, you should only worry about master or creating a branch off of master. This sometimes allows me to merge changes and fix/add things before it goes live on the site. It looks right now that you are merging a local copy of the gh-pages branch into this repos master.

**Comment by tkraak** [comment]
@rtorr I got you now, makes total sense and I like your workflow / thinking. Learned something!

Give me a few hours to issue proper pull requests from master.

**Comment by rtorr** [comment]
@tkraak you rock! thanks :)

**Comment by tkraak** [comment]
@rtorr Something still does not seem right with the new PR I just submitted. Why does it include 21 commits? Looks like I have to start over from the VERY beginning?!

**Comment by rtorr** [comment]
@tkraak it looks like at some point your branch pulled or was copied from gh-pages branch https://github.com/tkraak/vim-cheat-sheet/network :( 

I know this is frustrating. I could merge it and be done with it for you :)

**Comment by rtorr** [comment]
@tkraak @suchtie merged :dancer: 

https://github.com/rtorr/vim-cheat-sheet/pull/115

**Comment by tkraak** [comment]
@rtorr I'm 100% sure it's due to me messing with gh-pages early on :)

All good now since it was a clean merge on your end either way.

I'll start over with a new fork and new local repo for further contributions ... thanks!

**Comment by rtorr** [comment]
@tkraak no worries! I knew it would work, just adds a lot of merge commits. I am happy for your help. 

Thanks again.
